source/includes/ios/_ios.md

```diff
@@ -1,8 +1,39 @@
 # iOS SDK
 
-## Introduction
+## iOS Device Compatibility
 
-### SDK Module Command Protocol
+The iOS version of the **SensumSDK** can be installed on devices with **v10.3.3 and up**.
+
+We recommend the **iPhone 5 and up** as devices to use with the SDK.
+
+## Bluetooth Device Compatibility
+
+* The iOS SDK supports connecting to BLE devices for reading heart rate measurements. For a list of tested compatible devices please view the <a href = "http://help.sensum.co/knowledge_base/topics/what-type-of-sensors-can-i-use"> list of compatible devices</a> at our Knowledge Centre.
+
+**Note:** This document is regularly updated with new devices. Please contact us for integration details. GSR data is only accessible from Shimmer devices at present.
+
+## Accepted Biometric Data Inputs
+
+|Metric Name|Unit|
+|-----------|----|
+|<a href = "http://help.sensum.co/knowledge_base/topics/why-measure-heart-rate">Heart Rate</a>|bpm|
+|<a href = "http://help.sensum.co/knowledge_base/topics/what-is-gsr">GSR</a>|Siemens<sup>*</sup>|
```

Comment:
IOS Doesn't do GSR, so *either* this and the android section could be deduplicated and cross linked *or* this GSR gets cut out.